commons/src/main/java/io/aiven/kafka/connect/common/source/input/JsonTransformer.java

```diff
@@ -87,26 +96,14 @@ public boolean doAdvance(final Consumer<? super byte[]> action) {
                         }
                     }
                     line = line.trim();
-                    action.accept(line.getBytes(StandardCharsets.UTF_8));
+                    action.accept(jsonConverter.toConnectData(offsetManagerEntry.getTopic(),
+                            line.getBytes(StandardCharsets.UTF_8)));
                     return true;
                 } catch (IOException e) {
                     LOGGER.error("Error reading input stream: {}", e.getMessage(), e);
```

Comment:
If this fails here with an IOException, is that due to the fact that their is nothing left in the iterator?
Just wondering if we should rethrow the exception in any case here, by checking if errors.tolerance is configured to 'none' or if that is overkill here.